collector/src/cgroups_cpuacct.cpp

```diff
@@ -361,6 +361,13 @@ void CMonitorCgroups::sample_cpuacct(double elapsed_sec)
     if ((m_pCfg->m_nCollectFlags & PK_CGROUP_CPU_ACCT) == 0)
         return;
 
+#ifdef PROMETHEUS_SUPPORT
+    if (m_pOutput->is_prometheus_enabled()) {
+        size_t size = sizeof(prometheus_kpi_cgroup_cpu) / sizeof(prometheus_kpi_cgroup_cpu[0]);
+        m_pOutput->init_prometheus_kpi(prometheus_kpi_cgroup_cpu, size);
```

Comment:
I was expecting this call, init_prometheus_kpi(), to be located inside CMonitorCgroups::init_cpuacct() since that should be done only once at the start of the application right?
Why do we repeat this call here every time we sample the CPU acct cgroup?